packages/kuma-gui/src/app/connections/views/ConnectionInboundSummaryClustersView.vue

```diff
@@ -9,53 +9,56 @@
       connection: '',
     }"
     name="connection-inbound-summary-clusters-view"
-    v-slot="{ route }"
+    v-slot="{ route, uri }"
   >
     <RouteTitle
       :render="false"
       :title="`Clusters`"
     />
     <AppView>
       <DataLoader
-        :src="`/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/data-path/clusters`"
-        v-slot="{ data: clusters, refresh }: ClustersDataSource"
+        :src="uri(sources, '/meshes/:mesh/dataplanes/:name/clusters', {
+          mesh: route.params.mesh,
+          name: route.params.dataPlane,
+        })"
+        v-slot="{ data , refresh }"
       >
-        <DataCollection
-          :items="clusters!.split('\n')"
-          :predicate="item => item.startsWith(`${props.data.service}::`)"
-          v-slot="{ items: lines }"
+        <template
+          v-for="prefix in [route.params.connection.replace('_', ':')]"
+          :key="typeof prefix"
```

Comment:
Just a nit because there is only one item in the loop, but I think using `typeof` can easily lead to non-unique keys. So do we need a `key` at all? Can it be the `prefix` itself or maybe the `connection`?

Reply:
> So do we need a key at all?

Yeah, unfortunately we do because of the linter 😭 . If we didn't have the linter I wouldn't do this, but in some case with `for` loops the `key` is important.

Moreover, in order to satisfy the linter you need to use a variable from the for statement itself.

I found this trick a while back, in the case of where you specifically don't want Vue to use its detach/reattach behaviour in a for loop (instead you want it to just change the properties of existing components), you can use `:key="typeof variableFromTheLoop"`. This means:

1. `key` will never change, so Vue will never attach/detach, and instead reset properties of existing components if the data changes.
2. We are using a variable from the for loop so this satisfies the linter.

I only found this out a while ago, so there are probably loads of places where we should use this form to prevent unwanted `mount/unmount` hooks being called unnecessarily.

Let me know if you have more questions around this, I not its a little strange looking, I can try and find the PR where this was super important - I remember also making a note on that PR so that could be useful.

Reply:
Ah here we go (easier to find than I thought 😄 ), take a look at the description https://github.com/kumahq/kuma-gui/pull/2878

Reply:
Actually, I wonder if I could add this info somewhere in our engineering docs 🤔

Reply:
Hmmm, I see 🤔 I agree that we shouldn't disable the rule on a global level, but probably a simple eslint-disable directive would have worked aswell? Would mean less confusion 😄

Reply:
I'll approve and unblock this 🙂